src/indicators/flooding/glofas/utils.py

```diff
@@ -250,13 +251,14 @@ def get_crps(
             observations = ds_reanalysis[station].reindex(
                 {"time": forecast.time}
             )
+
             if normalization == "mean":
                 norm = observations.mean().values
             elif normalization == "std":
                 norm = observations.std().values
             elif normalization is None:
```

Comment:
You can take out this last else if because if it's not met, then `None` will be passed to the crps method